[2025-06-11T06:50:02.079+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-11T06:50:02.102+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: even_odd_branching.branching scheduled__2025-06-11T06:45:00+00:00 [queued]>
[2025-06-11T06:50:02.111+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: even_odd_branching.branching scheduled__2025-06-11T06:45:00+00:00 [queued]>
[2025-06-11T06:50:02.111+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-06-11T06:50:02.127+0000] {taskinstance.py:2330} INFO - Executing <Task(BranchPythonOperator): branching> on 2025-06-11 06:45:00+00:00
[2025-06-11T06:50:02.138+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1152) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-06-11T06:50:02.139+0000] {standard_task_runner.py:63} INFO - Started process 1154 to run task
[2025-06-11T06:50:02.141+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'even_odd_branching', 'branching', 'scheduled__2025-06-11T06:45:00+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/even_odd_branching.py', '--cfg-path', '/tmp/tmp7bdhj4e9']
[2025-06-11T06:50:02.145+0000] {standard_task_runner.py:91} INFO - Job 45: Subtask branching
[2025-06-11T06:50:02.191+0000] {task_command.py:426} INFO - Running <TaskInstance: even_odd_branching.branching scheduled__2025-06-11T06:45:00+00:00 [running]> on host 84435f84cb56
[2025-06-11T06:50:02.274+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='even_odd_branching' AIRFLOW_CTX_TASK_ID='branching' AIRFLOW_CTX_EXECUTION_DATE='2025-06-11T06:45:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-06-11T06:45:00+00:00'
[2025-06-11T06:50:02.276+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-11T06:50:02.276+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2025-06-11T06:50:02.277+0000] {python.py:237} INFO - Done. Returned value was: even_task
[2025-06-11T06:50:02.277+0000] {branch.py:36} INFO - Branch into even_task
[2025-06-11T06:50:02.277+0000] {skipmixin.py:178} INFO - Following branch even_task
[2025-06-11T06:50:02.295+0000] {skipmixin.py:246} INFO - Skipping tasks [('odd_task', -1)]
[2025-06-11T06:50:02.322+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-11T06:50:02.346+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=even_odd_branching, task_id=branching, run_id=scheduled__2025-06-11T06:45:00+00:00, execution_date=20250611T064500, start_date=20250611T065002, end_date=20250611T065002
[2025-06-11T06:50:02.393+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-11T06:50:02.448+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-06-11T06:50:02.450+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
